src/routes/AppRouter.js

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
-import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';
-import {Products} from '../pages/Products';
-import {Cart} from '../pages/Cart';
-import {Home} from '../pages/Home';
-import {Header} from '../components/Header';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { Products } from '../pages/Products';
+import { Cart } from '../pages/Cart';
+import { Home } from '../pages/Home';
+import { Header } from '../components/Header';
 import { Login } from '../pages/Login';
 
 export const AppRouter = () => {
   return (
     <Router>
       <Header />
       <Switch>
-        <Route exact path='/'>
+        <Route exact path="/">
           <Home />
         </Route>
-        <Route path='/products'>
+        <Route path="/products">
           <Products />
         </Route>
-        <Route path='/cart'>
+        <Route path="/cart">
           <Cart />
         </Route>
-        <Route path='/login'>
+        <Route path="/login">
```

Comment:
What do you think about creating a file called routes.js and in this file?, we could add a variable "routes"  and export it as default then in AppRoutes.js import this file and create a map with each Route.

Example: 
export const routes = [
{ path:'/', element={ Home }, exact={true} },
{ path:'/cart', element={ Cart } }
...
]

Reply:
Sounds good. I didn't touch any of the files you gave us except for the Products module; the others got modified because of prettier.